website/content/docs/enterprise/index.mdx

```diff
@@ -8,22 +8,73 @@ description: >-
+
+#### Operational Simplification
+- [Automated Upgrades](/docs/enterprise/upgrades): ease upgrades by automating the transition from existing to newly deployed Consul servers
+- [Consul-Terraform-Sync Enterprise](/docs/nia/enterprise): leverage the enhanced network infrastructure automation capabilities of the enterprise version of Consul-Terraform-Sync
+
+#### Complex Network Topology Support
+- [Network Areas](/docs/enterprise/federation): support complex network topologies between federated Consul datacenters with pairwise federation rather than full mesh federation
+- [Network Segments](/docs/enterprise/network-segments): support complex network topologies within a Consul datacenter by enforcing boundaries in Consul client gossip traffic
+
+#### Governance
+- [OIDC Auth Method](/docs/security/acl/auth-methods/oidc): manage user access to Consul through an OIDC identity provider instead of Consul ACL tokens directly
+- [Audit Logging](/docs/enterprise/audit-logging): understand Consul access and usage patterns by reviewing access to the Consul HTTP API 
+- [Sentinel for KV](/docs/enterprise/sentinel): policy-as-code framework for defining advanced key-value storage access control policies
+
+# Gain Access to Enterprise Features
+
+### Self-Managed Consul
+
+To gain access to Consul Enterprise features in a self-managed installation,
+[apply a purchased license](/docs/enterprise/license/overview)
+to the Consul Enterprise binary that grants access to the desired features.
+
+### HCP Consul
+
+No action is required to gain access to Consul Enterprise features in a
+HashiCorp Cloud Platform installation.
+
+# Consul Enterprise Feature Availability
+
+Consul Enterprise is offered in several forms that provide a different mix of enterprise features.
+Choose the form that best suits the needs of your organization.
+
+Available enterprise features per Consul form and license include:
+
+| Feature                                                  | [HashiCorp Cloud Platform (HCP) Consul]                 | [Consul Enterprise] | Legacy Consul Enterprise (module-based)           |
+| -------------------------------------------------------- | ------------------------------------------------------- | ------------------- | ------------------------------------------------- |
+| Consul servers as a managed service                      | Yes                                                     | No (self-managed)   | No (self-managed)                                 |
+| [Admin Partitions](/docs/enterprise/admin-partitions)    | Standard tier and above                                 | Yes                 | With Governance and Policy module                 |
+| [Namespaces](/docs/enterprise/namespaces)                | Standard tier and above                                 | Yes                 | With Governance and Policy module                 |
```

Comment:
@jkirschner-hashicorp I like the table. The one piece I am not a fan of are the empty slots under HCP. I understand that some of these features are not applicable to HCP but we should indicate as such. This will prevent customers asking questions about these features in HCP.

Reply:
Thanks @karl-cardenas-coding. Just synced with @srikrishmurthy on this. I'll make a few changes (one of these features might be truly "Not Applicable" in the case of HCP Consul).